.github/workflows/scripts/validate-coderabbit.yml

```
@@ -0,0 +1,54 @@
+      github_token:
+        required: true
+        type: string
+        
+jobs:
+    CodeRabbit-Approval:
+        name: Validate CodeRabbit.ai Approval
+        runs-on: ubuntu-latest
+        needs : [Code-Quality-Checks, Test-Application, Start-App-Without-Docker, Docker-Start-Check]
+        steps:
+        - name: Check CodeRabbit.ai Approval
+            run: |
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 17: mapping values are not allowed in this context

(syntax-check)